docker-app/qfieldcloud/core/cron.py

```diff
@@ -135,10 +137,18 @@ def do(self):
                     storage.delete_stored_package(project, package_id)
 
             else:
-                delete_count = File.objects.filter(
-                    project=project,
-                    package_job_id__in=job_ids,
-                ).delete()
+                delete_count = (
+                    File.objects.filter(
+                        project=project,
+                        package_job_id__in=job_ids,
+                    )
+                    .exclude(
+                        package_job_id__in=project.last_package_jobs().values_list(
+                            "id", flat=True
+                        )
```

Comment:
```suggestion
                        package_job_id__in=latest_project_package_jobs_id_qs,
```

I think this is slightly more redable if you move the qs to a variable.